snapshots/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+*/
+!.gitignore
```

Comment:
Why is this needed ? @bijanmmarkes

Reply:
It probably is a relic of Modeify as a functional TDM solution.  if removing it doesn't impact anything lets get rid of it.  @bijanmmarkes @linuxwebexpert

Reply:
agreed

Reply:
In the original .gitignore, It would not let me just ignore the files and keep an empty folder. It required me to create a .gitignore within the actual folder snapshots or it would either have to be completely ignored or uploaded with all my local snapshots. @linuxwebexpert